src/makeCalculator.js

```diff
@@ -33,11 +33,63 @@
  *
  * calculator.result === -28
  *
- *
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
```

Comment:
Here you need to make not just usual **let** but you need **this.result = callback(this.result, number);**

You have some missanderstandings in this task. I would recommend you to ask in FE chat!